### PR TITLE
Fix missing details in rebalance performance tuning table

### PR DESCRIPTION
**Pull request by tomncooper**
### Type of change

Documentation

### Description

Fixes missing details for server-side Cruise Control replica movement strategy settings.

### CODE REVIEW

File: documentation/modules/cruise-control/con-rebalance-performance.adoc [hunk 1]
```diff
@@ -74,7 +74,7 @@ The relevant configurations are summarized below:
 | `replicationThrottle`
 
 | `default.replica.movement.strategies`             .2+| 
-  The list of strategies (in priority order) used to determine the order in which partition reassignment commands are executed for generated proposals. Use a comma separated string for server and a YAML array for `KafkaRebalance` resources.
+  The list of strategies (in priority order) used to determine the order in which partition reassignment commands are executed for generated proposals. For the sever setting, use a comma separated string with the fully qualified names of the strategy class (add `com.linkedin.kafka.cruisecontrol.executor.strategy.` to the start of each class name). For the for `KafkaRebalance` resource setting use a YAML array of strategy class names.
```
Comment (ppatierno) [inline documentation/modules/cruise-control/con-rebalance-performance.adoc:77]: typo "server" and double "for" (For the for `KafkaRebalance`)

**Review by ppatierno — APPROVED** [review]
There are a couple of typos to fix but lgtm (after fixing them :-))

File: documentation/modules/cruise-control/con-rebalance-performance.adoc [hunk 2]
```diff
@@ -74,7 +74,7 @@ The relevant configurations are summarized below:
 | `replicationThrottle`
 
 | `default.replica.movement.strategies`             .2+| 
-  The list of strategies (in priority order) used to determine the order in which partition reassignment commands are executed for generated proposals. Use a comma separated string for server and a YAML array for `KafkaRebalance` resources.
+  The list of strategies (in priority order) used to determine the order in which partition reassignment commands are executed for generated proposals. For the server setting, use a comma separated string with the fully qualified names of the strategy class (add `com.linkedin.kafka.cruisecontrol.executor.strategy.` to the start of each class name). For the `KafkaRebalance` resource setting use a YAML array of strategy class names.
```
Comment (laidan6000) [inline documentation/modules/cruise-control/con-rebalance-performance.adoc:77]: Perhaps start a new paragraph at "For the server setting". The table cell is quite lengthy now!

**Review by laidan6000 — APPROVED** [review]
Looks good to me. One formatting suggestion.

**Pull request merged**